src/com/owncloud/android/lib/resources/files/ChunkedUploadRemoteFileOperation.java

```diff
@@ -48,41 +51,49 @@
     private static final String OC_CHUNKED_HEADER = "OC-Chunked";
     private static final String OC_CHUNK_SIZE_HEADER = "OC-Chunk-Size";
     private static final String TAG = ChunkedUploadRemoteFileOperation.class.getSimpleName();
-
-    public ChunkedUploadRemoteFileOperation(String storagePath, String remotePath, String mimeType){
-        super(storagePath, remotePath, mimeType);
-    }
+    private Context mContext;
 
     public ChunkedUploadRemoteFileOperation(
-            String storagePath, String remotePath, String mimeType, String requiredEtag
+            String storagePath, String remotePath, String mimeType, String requiredEtag, Context context
```

Comment:
As @przybylski mentioned in one of my PRs, as a general rule the context should always be the first parameter :)